src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIBookingService.kt

```diff
@@ -70,8 +74,8 @@ class CommunityAPIBookingService(
       appointmentStart = appointmentTime,
       appointmentEnd = appointmentTime.plusMinutes(durationInMinutes.toLong()),
       officeLocationCode = officeLocation,
-      notes = getNotes(referral, resourceUrl, "Appointment"),
-      countsTowardsRarDays = true, // Fixme: For assessment booking this should be false and will pass in when assessment booking is done
+      notes = getNotes(referral, resourceUrl, "${get(appointmentType, notesFieldQualifier)} Appointment"),
+      countsTowardsRarDays = get(appointmentType, countsTowardsRarDays),
```

Comment:
I was under the impression that `countsTowardsRarDays` was already mapped in the structures of the Delius reference data; do we need it here?

The concept of "counts towards RAR days" is something our domain shouldn't need to understand or configure. It's the difference between "delivery" and "non-delivery" appointments that matters in the domain, and we can pass that along, right?

Reply:
I think you're probably right that community-api can determine this based on being told whether we are booking for a supplier assessment or a service delivery but this was unclear at the start. I can raise a tech debt ticket to cover this.